packages/url/src/get-query-args.js

```diff
@@ -85,7 +85,12 @@ export function getQueryArgs( url ) {
 					// Filtering avoids decoding as `undefined` for value, where
 					// default is restored in destructuring assignment.
 					.filter( Boolean )
-					.map( decodeURIComponent );
+					.map( ( kv ) => {
+						try {
+							return decodeURIComponent( kv );
+						} catch ( uriComponentError ) {}
+						return kv;
+					} );
```

Comment:
As @ntsekouras also brought up, we should be able to use `safeDecodeURIComponent()` here since it does exactly what we're aiming for.

```suggestion
					.map( safeDecodeURIComponent );
```

Note that we'll need to import it in this module.